modules/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -311,10 +311,10 @@ error.undefined.transformer=\
   undefined transformer ''{0}''
 
 error.transformer.invalid.output.usage=\
-  invalid usage of variable ''{0}'': outputs cannot be used in rhs expressions inside transformer block
+  invalid usage of variable ''{0}'': target cannot be used in rhs expressions inside transformer block
 
 error.transformer.invalid.input.update=\
-  invalid usage of variable ''{0}'': inputs cannot be updated inside transformer block
+  invalid usage of variable ''{0}'': source cannot be updated inside transformer block
```

Comment:
Shouldn't this be source or parameters? @SupunS

Reply:
yeah, added the changes via https://github.com/ballerinalang/ballerina/pull/4060